WordPress/Learn
# Standardise taxonomies and meta fields across post types

See #1009 

This PR sets up the new taxonomies we will be using across all post types:

1. A new `wporg_language` taxonomy to be populated with all the possible locales
2. Extends the `topic` taxonomy (currently only applied to tutorials) to all post types

Once these are in place we can start migrating all the post content to use them:

- Lesson plan `category` -> `topic`
- Tutorial `video_language` meta field -> `wporg_language`
- Course `course-category` -> `topic`

After that the old taxonomies and meta fields can be removed (or possibly hidden from the UI if there is any value in keeping them around).

A number of frontend tasks will then follow to switch filters and queries to use the new taxonomies.

### Screenshots

#### Admin tables
| Lesson Plans | Languages |
|-|-|
| ![Screen Shot 2022-10-20 at 2 16 16 PM](https://user-images.githubusercontent.com/1017872/196833974-67b2b2df-517d-4e7b-bab1-ad8b053daca1.jpg) | ![Screen Shot 2022-10-20 at 2 16 43 PM](https://user-images.githubusercontent.com/1017872/196834015-5f27eb25-c0eb-4046-88f5-0748f2982511.jpg) | 

#### Posts

| Lesson Plan | Tutorial |
|-|-|
| ![Screen Shot 2022-10-20 at 2 25 20 PM](https://user-images.githubusercontent.com/1017872/196834527-23040ffb-ef68-4f13-9425-76a9c6eeb144.jpg) | ![Screen Shot 2022-10-20 at 2 26 26 PM](https://user-images.githubusercontent.com/1017872/196834720-69a83ad1-94ef-403f-85b0-7a7e3b2c9e58.jpg) | 

### How to test

1. Load the admin and check that Lesson Plans, Tutorials, Meetings and Courses all have Topic and Language taxonomies.
2. Edit different post types adding Languages (see screenshots above for proposed data), and Topics (should already be plenty of terms from Tutorials).
3. Smoke test frontend after adding new terms to post content.
